### PR TITLE
8 get api reviews category

### PR DESCRIPTION
**Pull request by fuzzyJess**
Completed functions to return reviews by categories with all tests passing.

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -70,7 +70,55 @@ describe("GET requests", () => {
                 })
         })
     })
+    describe("8.GET /api/reviews/?sort_by=dexterity", () => {
+        test("status: 200, responds with array of one review object when category provided only matches one", () => {
+            return request(app)
+                .get("/api/reviews/?sort_by=dexterity")
```
Comment (HeatherEver) [inline __tests__/app.test.js:76]: please can you change `sort_by` to `category`

File: __tests__/app.test.js [hunk 2]
```diff
@@ -70,7 +70,55 @@ describe("GET requests", () => {
...
+                .get("/api/reviews/")
+                .expect(200)
+                .then(({ body }) => {
+                    const { reviews } = body;
+                    expect(reviews).toHaveLength(13);
+                    expect(reviews).toBeInstanceOf(Array);
+                  //  expect(reviews[0]).toEqual()
+                })
+        })
+    })
```
Comment (HeatherEver) [inline __tests__/app.test.js:119]: your tests are good but they're not actually testing the order they have come back in - we can use jest sorted to check this and just add in another expect statement!

File: models/reviews.models.js [hunk 3]
```diff
@@ -48,7 +48,51 @@ exports.updateReview = (id, votes) => {
...
+            'strategy', 
+            'social deduction',
+            'hidden-roles', 
+            'dexterity', 
+            'push-your-luck', 
+            'roll-and-write',
+            'deck-building', 
+            'engine-building',
+            'euro game',
+            "children's games"
```
Comment (HeatherEver) [inline models/reviews.models.js:74]: this is fine and will work well, however something to note for the future if you want to refactor this we could query the database here for the categories - just soemthing to note!

**Review by HeatherEver — COMMENTED** [review]
lovely work jessica! just one thing to change in your tests so your using category instead of sort_by

File: __tests__/app.test.js (hunk 1 above)
Comment (fuzzyJess) [reply __tests__/app.test.js:76]: Ah - yes, have managed that now

**Pull request merged**